lib/membrane_rtc_engine/engine.ex

```diff
@@ -268,7 +268,7 @@ defmodule Membrane.RTC.Engine do
   @type track_ready_action_t() ::
           {:notify,
            {:track_ready, Track.id(), Track.encoding(),
-            depayloading_filter :: Membrane.ParentSpec.child_spec_t()}}
+            depayloading_filter :: Membrane.ParentSpec.child_spec_t() | nil}}
```

Comment:
Please write when `depayloading_filter` should be `nil` and what happens when track is in `[:raw]` format only but `depayloading_filter` is not `nil`